assets/Zed/js/modules/form-toggler.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+'use strict';
+
+$(document).ready(function() {
+
+    createToggleElements();
+
+    $('.toggle-trigger').each(function (index, selectItem) {
+        var group = $(selectItem).data('toggle-group');
+        var selectedValue = selectItem.value;
+        var items = getItems(group);
+        var activeItem = getActiveItem(group, selectedValue);
+
+        toggleItems(items, activeItem);
+
+        $(selectItem).on('change', function (event) {
+            selectedValue = event.target.value;
+            activeItem = getActiveItem(group, selectedValue);
+
+            toggleItems(items, activeItem);
+        });
+    });
+
+    function createToggleElements() {
+        $('.toggle-inner-item').parent('.form-group').addClass('toggle-item');
```

Comment:
You creating nothing in this method. Please move this login into `initToggleElements`.

Reply:
Fixed